.github/workflows/update-cmd-repositories.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: Update cmd-nsmgr
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  update-cmd-nsmgr:
+    name: Update cmd-nsmgr
+    runs-on: ubuntu-latest
+    if: github.repository == 'networkservicemesh/cmd-template'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: [networkservicemesh/cmd-nsmgr, networkservicemesh/cmd-forwarder-sriov, networkservicemesh/cmd-forwarder-vppagent, networkservicemesh/cmd-registry-proxy-dns, networkservicemesh/cmd-registry-memory, networkservicemesh/cmd-exclude-prefixes-k8s]
```

Comment:
Do we need to start with one repository to check how it will work? :)

Reply:
Lets definitely start with one and go from there :)

Reply:
So... how do we avoid here having .github/workflows/update-cmd-repositories.yaml pushed out to all the cmd-* repos?

Reply:
> how do we avoid here having .github/workflows/update-cmd-repositories.yaml pushed out to all the cmd-* repos?

I did not found an easy solution for this, can we just still use check that this action will work only if the repository name is has 'template'?

Reply:
> Lets definitely start with one and go from there :)

Done.